app/views/content/life-as-a-teacher/explore-subjects/_categories.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="row">
+  <section class="category col-space-m">
+    <section class="container category__cards main-section">
+      <nav class="category__nav-cards">
+        <ul>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Computing",
+              description: "Discover how computing is vital, helping children learn digital skills they'll need throughout life.",
+              path: "/life-as-a-teacher/explore-subjects/computing",
+            )) %>
+          <%= render Categories::CardComponent.new(heading_tag: "h2", card:
+            OpenStruct.new(
+              title: "Maths",
+              description: "Find out how to become a maths teacher and show children the importance of maths in everything we do.",
```

Comment:
Maybe change to 'Find out how to become a maths teacher, so you can show children how important maths is in everyday life and in understanding the world.' Or something similar?

Just think the 'we' sounds like it could refer to everything we do as teachers, if that makes sense!

Reply:
```suggestion
              description: "Find out how to become a maths teacher, so you can show children how important maths is in everyday life and in understanding the world.",
```